tests/foreman/ui/test_repositories.py

```diff
@@ -0,0 +1,58 @@
+"""Unit tests for the new ``repositories`` paths.
+
+:Requirement: Repository
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Repositories
+
+:team: Phoenix-content
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
+def test_positive_custom_products_disabled_by_default(
+    session,
+    default_location,
+    setup_content,
+    rhel_contenthost,
+    target_sat,
+):
+    """Verify that custom products should be disabled by default for content hosts
+    and activation keys
+
+    :id: 05bdf790-a7a1-48b1-bbae-dc25b6ee7d58
+
+    :steps:
+        1. Create custom product and upload repository
+        2. Attach to activation key
+        3. Register Host
+        4. Assert that custom proudcts are disabled by default
+
+    :expectedresults: Custom products should be disabled by default.
```

Comment:
Can you link any BZ/RFE which introduces this change here?

Reply:
Its just on RFE... DO we normally attach jira card numbers ?

Reply:
No, your jira has BZ linked to it, can we link that bz? https://bugzilla.redhat.com/show_bug.cgi?id=1265120